vassal-app/src/main/java/VASSAL/build/module/map/Flare.java

```diff
@@ -0,0 +1,343 @@
+/*
+ * Copyright (c) 2020 Vassalengine.org   Brian Reynolds
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License (LGPL) as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public
+ * License along with this library; if not, copies are available
+ * at http://www.opensource.org.
+ */
+
+package VASSAL.build.module.map;
+
+import java.awt.event.MouseEvent;
+import java.awt.event.ActionEvent;
+import VASSAL.build.module.documentation.HelpFile;
+import VASSAL.command.Command;
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.Graphics;
+import VASSAL.build.GameModule;
+import VASSAL.build.Buildable;
+import javax.swing.Timer;
+
+import org.apache.commons.lang3.SystemUtils;
+
+import java.awt.Point;
+import VASSAL.build.module.Map;
+import java.awt.event.MouseListener;
+import java.awt.event.ActionListener;
+import VASSAL.build.module.GameComponent;
+import VASSAL.build.module.GlobalOptions;
+import VASSAL.command.CommandEncoder;
+import VASSAL.build.AbstractConfigurable;
+import VASSAL.build.AutoConfigurable;
+import VASSAL.command.FlareCommand;
+import VASSAL.configure.ColorConfigurer;
+import VASSAL.configure.StringEnum;
+import VASSAL.i18n.Resources;
+import VASSAL.tools.swing.SwingUtils;
```

Comment:
Sort the JDK imports into one group, ours into another.

Reply:
fixed